modules/onetagBidAdapter.js

```diff
@@ -6,13 +6,7 @@ const ENDPOINT = 'https://onetag-sys.com/prebid-request';
 const USER_SYNC_ENDPOINT = 'https://onetag-sys.com/usync/';
 const BIDDER_CODE = 'onetag';
 const BANNER = 'banner';
```

Comment:
You can import the mediaType (both, BANNER and VIDEO) from `src/mediaTypes.js` file. Please check the appnexusBidAdapter.js for more details.

Reply:
@OneTagDevOps

Reply:
@Fawke just updated

Reply:
Thanks!